src/Bridge/TracerBridge.php

```diff
@@ -28,5 +32,8 @@ public static function getSubscribedEvents(): array
     public function onTerminate(): void
     {
         $this->tracer->flush();
```

Comment:
@allflame Should we have reset the state inside `flush()` method implementation in tracer?

Reply:
Discussed with @allflame.
From some points of view - `flush` can have semantics `send spans, but not reset fully state`.

Let's don't change internal behavior for `flush` internal logic - but implement logic from the outside  (e.g. https://github.com/code-tool/jaeger-client-symfony-bridge/pull/28#discussion_r1286961747)